### PR TITLE
fixed deps in next-tinacms-cloudinary

### PR DESCRIPTION
**Pull request by logan-anderson**
pretty sure it was installing its own version of TinaCMS which was causing issues. (it was a direct dependency)

Right now when I am installing it in my blog I get this error.

"can not resolve module react-final-form"

### CODE REVIEW

File: packages/next-tinacms-cloudinary/package.json [hunk 1]
```diff
@@ -13,33 +13,49 @@
...
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-is": "^17.0.2",
-    "tinacms": "0.31.0",
-    "tsup": "^3.7.0",
-    "typescript": "^3.8.3"
+    "tsup": "^3.7.0"
   },
   "devDependencies": {
+    "@tinacms/core": "^0.31.0",
```
Comment (jeffsee55) [inline packages/next-tinacms-cloudinary/package.json:20]: I'm assuming you got yarn complaining about these, we had to do the same thing over [here](https://github.com/tinacms/tina-graphql-gateway/blob/main/packages/tina-graphql-gateway/package.json#L47-L53). Not ideal but looks good to me

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "license:add": "$PWD/scripts/add-license.sh",
     "license:delete": "$PWD/scripts/delete-license.sh",
     "ci:version-check": "yarn version check",
-    "ci:build": "yarn workspaces foreach --exclude=sc --exclude=tina-cloud-starter --exclude=tina-tailwind-sidebar-demo  --exclude=test --exclude=demo -pt run build",
+    "ci:build": "yarn workspaces foreach --exclude=sc --exclude=tina-cloud-starter --exclude=tina-tailwind-sidebar-demo  --exclude=test --exclude=demo -pt --topological-dev run build",
```
Comment (jeffsee55) [inline package.json:18]: This _should_ be covered already with `-pt` (`t` being the `topological-dev` shorthand), was that still not working for you?

**Review by jeffsee55 — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (logan-anderson) [reply package.json:18]: I dont think it is. https://yarnpkg.com/cli/workspaces/foreach#options
`-t` or `--topological`: `Run the command after all workspaces it depends on (regular) have finished`
while
`--topological-dev`: does the same thing for dev dependencies.  I was getting a build error when not using ` --topological-dev`

File: packages/next-tinacms-cloudinary/package.json (hunk 1 above)
Comment (logan-anderson) [reply packages/next-tinacms-cloudinary/package.json:20]: I removed them and did not get any errors.

File: package.json (hunk 2 above)
Comment (logan-anderson) [reply package.json:18]: I for this PR to build I also need it to build the dev dependencies in order

Comment (jeffsee55) [reply package.json:18]: Ah gotcha, ok works for me

**Pull request merged**